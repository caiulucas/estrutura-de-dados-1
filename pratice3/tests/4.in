12
Estevao 115
Antônio 40
Betina 58
Thomas 90
Miguel 42
AnaVitoria 197
ArthurGabriel 130
Dominic 152
Luan 176
Chloe 140
Kauê 158
JoaoMiguel 119
