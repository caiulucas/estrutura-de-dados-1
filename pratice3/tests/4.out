Time 1
[L] AnaVitoria 197
[P] Chloe 140
[E] Thomas 90

Time 2
[L] Luan 176
[P] ArthurGabriel 130
[E] Betina 58

Time 3
[L] Kauê 158
[P] JoaoMiguel 119
[E] Miguel 42

Time 4
[L] Dominic 152
[P] Estevao 115
[E] Antônio 40

